app/include/zmk/split/bluetooth/central.h

```diff
@@ -4,6 +4,12 @@
 #include <zephyr/bluetooth/addr.h>
 #include <zmk/behavior.h>
 
+enum peripheral_slot_state {
```

Comment:
These should be properly namespaced if published externally like this, e.g. `zmk_split_bt_central_peripheral_slot_state`

Reply:
I'm also not convinced that the "slot" language should escape this, instead of using the more generic "source" concept used elsewhere.